packages/react-examples/src/react-divider/Divider/DividerStory.tsx

```diff
@@ -0,0 +1,17 @@
+import * as React from 'react';
+import { ax } from '@fluentui/react-make-styles';
+import * as classes from '../react-divider.stories.scss';
+
+export interface DividerStoryProps {
+  label?: string;
+  children?: any;
+  className?: string;
+}
+export const DividerStory = (props?: DividerStoryProps) => {
+  return (
+    <div className={ax(classes.item, props?.className)}>
+      <label>{props?.label}</label>
+      {props?.children}
+    </div>
+  );
+};
```

Comment:
Any specific reason to have this in a separate file and not just defined within `Divider.stories.tsx`?

Reply:
Updated